web_ui/frontend/components/FederationOverview.tsx

```diff
@@ -23,22 +24,38 @@ const LinkBox = ({href, text} : {href: string, text: string}) => {
     )
 }
 
+const UrlData = [
+    {key: ["Federation", "NamespaceUrl", "Value"], text: "Namespace Registry"},
+    {key: ["Federation", "DirectorUrl", "Value"], text: "Director"},
+    {key: ["Federation", "RegistryUrl", "Value"], text: "Registry"},
+    {key: ["Federation", "TopologyNamespaceUrl", "Value"], text: "Topology Namespace"},
+    {key: ["Federation", "DiscoveryUrl", "Value"], text: "Discovery"},
+    {key: ["Federation", "JwkUrl", "Value"], text: "JWK"}
+]
+
 const FederationOverview = () => {
 
-    const [config, setConfig] = useState<{ [key: string] : string | undefined} | undefined>(undefined)
+    const [config, setConfig] = useState<{text: string, url: string | undefined}[]>([])
 
     let getConfig = async () => {
         let response = await fetch("/api/v1.0/config")
```

Comment:
This call is not public, and it causes the director UI to jump to the login page
A better endpoint should be `/.well-known/pelican-configuration`, which will give you a json like the following:
```json
{
  "director_endpoint": "https://6ee28c5df997:8888",
  "namespace_registration_endpoint": "https://osdf-itb-registry.osg-htc.org",
  "jwks_uri": "https://6ee28c5df997:8888/.well-known/issuer.jwks",
  "broker_endpoint": ""
}

```

Reply:
@haoming29 Where can I find what the keys will be for the TopologyNamespaceURL and the DiscoveryURL?

Reply:
Good question. I don't think it's available at that endpoint, and that link is usually what `DiscoveryURL` will be pointed to. Do you want to me to prepare a new API for the UI or add the two keys to the `pelican-configuration` endpoint?

Reply:
That sounds good to me